scripts/continuous/fs/run.sh

```diff
@@ -127,7 +127,6 @@ interactive_tests() {
 	rm $PID_FILE
 }
 
-interactive_tests $EXPECT_TESTS_BASE/x86_fs_unit_tests.config
```

Comment:
may be just comment it?

Reply:
It was my proposition, should be reverted in the future. Not master seems to be broken anyway. I am going to disable all fs autotests temporary to figure out what's going on.

Reply:
@alexkalmuk Ok, thanks for the explanation